src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --another-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  position: relative;
+}
+
+.header__logo-link {
+  display: flex;
+  width: 40px;
```

Comment:
It's not a good idea to fix the size of a container unless it's necessary. It's better to let the content size dictate it. This helps to avoid overflow or an accidental scroll bar. So, if there is no such requirement, consider removing the fixed width and height.